conda-recipes/impyla/meta.yaml

```diff
@@ -25,6 +26,7 @@ requirements:
     - thrift_sasl
     - bitarray
     - thriftpy
+    - sasl >=0.2.1
```

Comment:
We can probably just remove them from the deps. Conda doesn't really do
soft dependencies. I suspect that in order to do hard dependencies we may
have to build a conda package for cyrus-sasl.
On Mon, Mar 21, 2016 at 23:51 Wes McKinney <notifications@github.com> wrote:

> In conda-recipes/impyla/meta.yaml
> <https://github.com/cloudera/ibis/pull/805#discussion_r56934252>:
>
> > @@ -25,6 +26,7 @@ requirements:
> >      - thrift_sasl
> >      - bitarray
> >      - thriftpy
> > +    - sasl >=0.2.1
>
> sasl and thrift_sasl are soft requirements of impyla -- I didn't see this
> before, but is there some way to do this with conda builds?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/cloudera/ibis/pull/805/files/0d0752a52b6ef4d6bc42b36b6c599f12d937f785#r56934252>
>